.github/workflows/pullrequest.yml

```
@@ -253,22 +253,14 @@ jobs:
         uses: runs-on/cache/restore@v4
         id: restore-cache
         with:
           path: |
-            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            ${{ needs.prepare.outputs.YARN_CACHE_FOLDER }}
             ${{ env.CYPRESS_CACHE_FOLDER }}
-          key: ${{ runner.os }}-deps-cypress-${{ hashFiles('yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-deps-cypress-
-            ${{ runner.os }}-
+          key: ${{ steps.set-cache-key.outputs.CACHE_KEY }}
```

actionlint (1.7.4)

property "set-cache-key" is not defined in object type {}

(expression)